### ISSUE TITLE
magidoc generate fails when using introspection type of `sdl`

### ISSUE THREAD

**Issue by BoilerTom90** [magidoc-org/magidoc#140]
### Describe the bug

I'm trying to use the following configuration in my magidoc.mjs config file:

```json
export default { 
   introspection: {
      type: 'sdl',
      paths: ['./src/generated/schema.graphql'],
   },
   // other fields omitted
}
```

The file ./src/generated/schema.graphql exists and contains our entire gql schema.

When I run the generate command, it fails with the following stack trace:

```bash
------- Stacktrace -------
Error: No paths found matching target glob patterns: [C:\GitProjectRepos\shure.system-api\sprint-q3.4-gqldoc\packages\standalone-server\src\generated\schema.graphql].
If you used relative paths, make sure the paths are relative to where the node command was launched or use absolute paths.
    at readFullSchema (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/@magidoc/rollup-plugin-parse-gql-schema/build/schema/parse.js:20:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async parseGraphqlSchema (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/@magidoc/rollup-plugin-parse-gql-schema/build/schema/parse.js:6:23) 
    at async parseSchema (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/@magidoc/rollup-plugin-parse-gql-schema/build/index.js:5:20)
    at async Object.executor (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/@magidoc/cli/build/tasks/all/loadGraphqlSchema.js:20:21)
    at async Task.run (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/listr2/dist/index.js:960:11)
    at async C:\GitProjectRepos\shure.system-api\sprint-q3.4-gqldoc\node_modules\p-map\index.js:57:22
```

I know the path is correct, and it is relative to where I'm invoking the generate command.  If I use the `file` option using the same relative path, as listed below, generate works.

```json
export default { 
   introspection: { 
      // this works.
      type: 'file',
      location: './src/generated/introspection.json',
   },
}
```

I wanted to use the schema file instead of the introspection file with the hope that the comments we've included in our schema file will show up as useful comments in the generated document. 

### Reproduction

you can use the information above

### Logs

```shell
Error: No paths found matching target glob patterns: [C:\GitProjectRepos\shure.system-api\sprint-q3.4-gqldoc\packages\standalone-server\src\generated\schema.graphql].
If you used relative paths, make sure the paths are relative to where the node command was launched or use absolute paths.
    at readFullSchema (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/@magidoc/rollup-plugin-parse-gql-schema/build/schema/parse.js:20:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async parseGraphqlSchema (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/@magidoc/rollup-plugin-parse-gql-schema/build/schema/parse.js:6:23) 
    at async parseSchema (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/@magidoc/rollup-plugin-parse-gql-schema/build/index.js:5:20)
    at async Object.executor (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/@magidoc/cli/build/tasks/all/loadGraphqlSchema.js:20:21)
    at async Task.run (file:///C:/GitProjectRepos/shure.system-api/sprint-q3.4-gqldoc/node_modules/listr2/dist/index.js:960:11)
    at async C:\GitProjectRepos\shure.system-api\sprint-q3.4-gqldoc\node_modules\p-map\index.js:57:22
```

### System Info

```shell
$ npx envinfo --system --binaries --browsers --npmPackages "{@magidoc/*}"
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

  System:
    OS: Windows 10 10.0.19042
    CPU: (12) x64 Intel(R) Core(TM) i7-10850H CPU @ 2.70GHz
    Memory: 4.99 GB / 31.73 GB
  Binaries:
    Node: 16.16.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 3.2.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 8.11.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 103.0.5060.134
    Edge: Spartan (44.19041.1266.0), Chromium (104.0.1293.47)
```

### Severity

serious, but I can work around it

**Comment by pelletier197** [comment]
Thanks for raising the issue. I will have a look as soon as possible!

**Comment by pelletier197** [comment]
I found the issue. It's due to a behaviour of `fast-glob` that I was not aware of (the library used for glob matching). It seems that backslashes are not supported for globs since they can be used to escape some characters. Fix should be simple and deployed in the next minutes. 

Thanks again for raising this issue.

**Comment by pelletier197** [comment]
Fixed with #141 and already deployed (2.7.1).  Enjoy!

**Comment by BoilerTom90** [comment]
Awesome. Thanks much!

I'll give it a try!
*~~~~~~~~~~~~~~~~~~~*
*Tom Hoffman*
